packages/client/src/runtime/core/model/applyModel.ts

```diff
@@ -78,8 +77,7 @@ function modelActionsLayer(client: Client, dmmfModelName: string): CompositeProx
     getPropertyValue(key) {
       const dmmfActionName = key as DMMF.ModelAction
 
-      let requestFn = (params: InternalRequestParams) => client._request(params)
-      requestFn = adaptErrors(dmmfActionName, dmmfModelName, client._clientVersion, requestFn)
+      const requestFn = (params: InternalRequestParams) => client._request(params)
```

Comment:
Maybe we can remove `requestFn` now to directly use `client._request` directly?

Reply:
We've decided to keep using `NotFoundError` for `*OrThrow` methods for now, so `adaptErrors` will be back soon